From bac4e61a56ec8d6a8977f95b3ca882c35ca1fe09 Tue, 24 Jan 2017 14:35:04 +0100
From: StefanoM <s.macone@libero.it>
Date: Tue, 24 Jan 2017 09:43:28 +0100
Subject: [PATCH] 


Change-Id: Iecfb9e53dcc0ae4823f9478518f3e1a260cc1378
Signed-off-by: StefanoM <s.macone@libero.it>
diff --git a/.gitignore b/.gitignore
index 83a69b9..ab297fc 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,26 +1,26 @@
-target/
-!.mvn/wrapper/maven-wrapper.jar
-
-### STS ###
-.apt_generated
-.classpath
-.factorypath
-.project
-.settings
-.springBeans
-
-### IntelliJ IDEA ###
-.idea
-*.iws
-*.iml
-*.ipr
-
-### NetBeans ###
-nbproject/private/
-build/
-nbbuild/
-dist/
-nbdist/
-.nb-gradle/
-/log.roo
-/bin/
+target/
+!.mvn/wrapper/maven-wrapper.jar
+
+### STS ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
+
+### NetBeans ###
+nbproject/private/
+build/
+nbbuild/
+dist/
+nbdist/
+.nb-gradle/
+/log.roo
+/bin/
diff --git a/src/main/java/it/istat/mec/mecbox/forms/UserCreateForm.java b/src/main/java/it/istat/mec/mecbox/forms/UserCreateForm.java
index 5d57cae..0870eba 100644
--- a/src/main/java/it/istat/mec/mecbox/forms/UserCreateForm.java
+++ b/src/main/java/it/istat/mec/mecbox/forms/UserCreateForm.java
@@ -27,14 +27,15 @@
    
 	 @NotNull
 	 @Size(min=2, max=30)
-    private String fullname;
+    private String Nome;
+	private String Cognome;
   
 
  
     private String role ;;
     
     public String toString() {
-        return "Person(Email: " + this.email + ", fullName: " + this.fullname + ")";
+        return "Person(Email: " + this.email + ", Nome: " + this.Nome + ", Cognome: " + this.Cognome + ")";
     }
 
 	public String getEmail() {
@@ -55,14 +56,25 @@
 
 	 
 
-	public String getFullname() {
-		return fullname;
+	public String getNome() {
+		return Nome;
 	}
 
-	public void setFullname(String fullname) {
-		this.fullname = fullname;
+	public void setNome(String Nome) {
+		this.Nome = Nome;
 	}
 
+	
+	public String getCognome() {
+		return Cognome;
+	}
+
+	public void setCognome(String Cognome) {
+		this.Cognome = Cognome;
+	}
+	
+	
+	
 	public String getRole() {
 		return role;
 	}
